content/chapter 3/3.3-sync.md

```
@@ -227,18 +227,19 @@ func main() {
 
 	// نویسنده
 	go func() {
-		for i := 0; i < 5; i++ {
-			smap.Write("name", fmt.Sprintf("value-%d", i))
-			time.Sleep(500 * time.Millisecond)
+		for i := range 5 {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)